uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/run.js

```diff
@@ -9,16 +9,22 @@ define([
     'tasks-assignments/vendor/text-angular',
     'tasks-assignments/modules/config',
     'tasks-assignments/modules/settings',
+    // TODO use only one router
```

Comment:
I have no idea. It was like this when I started working on that. But I'll refactor it when I work on certain modules.